Bottom: 3334ca6c260f2e9d73363416a3a7cd695db768b0
Top:    38fda2ea4c4070f1e8647e11801c3552b407051c
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-05 15:28:14 +0800

autofs-5.0.7 - fix incorrect var name in test

Fix incorrect variable name used for test of buffer size for getgrgid_r.


---

diff --git a/lib/mounts.c b/lib/mounts.c
index 550445c..0caa0aa 100644
--- a/lib/mounts.c
+++ b/lib/mounts.c
@@ -1222,7 +1222,7 @@ void set_tsd_user_vars(unsigned int logopt, uid_t uid, gid_t gid)
 	/* Try to get group info */
 
 	grplen = sysconf(_SC_GETGR_R_SIZE_MAX);
-	if (tmplen < 0) {
+	if (grplen < 0) {
 		error(logopt, "failed to get buffer size for getgrgid_r");
 		goto free_tsv_home;
 	}
